src/playbook.yaml

```diff
@@ -94,10 +94,6 @@
           echo 'LoadModule python_module /usr/lib/apache2/modules/mod_python.so' >> /etc/apache2/mods-available/python.load
         chdir: /tmp/mod_python_source
 
-    # Install python synapse client
-    - name: Install Python Synapse client
```

Comment:
could you explain why you are removing this now?

Reply:
The synapseclient is installed in a venv by reticulate/synapser.